day11/spiralMatrix.py

```diff
@@ -0,0 +1,58 @@
+def spiralMatrixArr(matrix):
+    direction = 0
```

Comment:
you don't do anything with this, which is a shame. i'm imagining a really cool solution that i'm too lazy to type out, but message me some time and i'll explain it, it has to do with python's built in handling of complex numbers and treating the matrix like a complex plane.

then i have an even more bullshit solution which is just 5 or so lines of recursive code which solves this really elegantly too.